spec/locale.html

```diff
@@ -35,11 +34,11 @@ <h1>Intl.Locale ( _tag_ [ , _options_ ] )</h1>
         1. Let _opt_ be a new Record.
         1. Let _calendar_ be ? GetOption(_options_, *"calendar"*, ~string~, ~empty~, *undefined*).
         1. If _calendar_ is not *undefined*, then
-          1. If _calendar_ does not match the Unicode Locale Identifier `type` nonterminal, throw a *RangeError* exception.
+          1. If _calendar_ cannot be matched by the <code>type</code> language tag nonterminal, throw a *RangeError* exception.
```

Comment:
same as above, please do not use "language tag" here while discussion the grammar